### DISCUSSION TITLE
when click alert section data undefinded

### DISCUSSION THREAD

**Discussion by arvinlp** [Security-Onion-Solutions/securityonion#13846] [Unsupported Versions]
Hi,
we have problem with open section alert, when request the page response data metrics is undefinded!
any solution have to solve this problem ?
![Untitled](https://github.com/user-attachments/assets/3d723e76-454f-49fe-9870-6188a07ad8c6)

**Comment by dougburks** [comment]
What version of Security Onion are you running?

Does Security Onion Console work correctly other than this one error message?

Does the Grid page show everything as OK?

What browser are you using?

Have you tried shift-reload in your browser or wiping the browser cache?

Have you tried an incognito window?

Have you tried a different browser?
